src/error/errorHandler.js

```diff
@@ -13,9 +16,11 @@ function handleError (err, req, res, next) {
 
   const responseCode = statusCodeResponses[err.message]
   if (responseCode) {
+    logger.warn(ERROR_MESSAGE, err)
     return res.status(responseCode).json({ errorMessage: err.message })
   }
 
+  logger.error(ERROR_MESSAGE, err)
```

Comment:
Please pay attention on this requirments:
![image](https://user-images.githubusercontent.com/52933810/146563195-e1e327ff-e9db-4788-993f-a80943240ee3.png)

You should log:

- method: `GET, POST .etc` -> `req.method`
- arguments: `/user, /suggest .etc` -> `req.originalUrl`
- error message

Reply:
Every method in controller isn't an HTTP method and we don't pass arguments to HTTP methods? I think it is about functions.

Reply:
Fixed